rust/toolchain.bzl

```diff
@@ -38,12 +150,17 @@ def _rust_toolchain_impl(ctx):
         default_edition = ctx.attr.default_edition,
         compilation_mode_opts = compilation_mode_opts,
         crosstool_files = ctx.files._crosstool,
+        libstd_and_allocator_ccinfo = _make_libstd_and_allocator_ccinfo(ctx, ctx.attr.rust_lib, ctx.attr.allocator_library),
     )
     return [toolchain]
 
 rust_toolchain = rule(
     implementation = _rust_toolchain_impl,
+    fragments = ["cpp"],
```

Comment:
Could we add a unit test that CcInfo generated on the rust toolchain orders rlibs in the way you expect? You'd have to create a new package under //test/unit, for inspiration, https://github.com/bazelbuild/rules_rust/pull/576 did something similar.